project_stage/README.rst

```diff
@@ -1,44 +1,42 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+.. |company| replace:: ADHOC SA
+
+.. |company_logo| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-logo.png
+   :alt: ADHOC SA
+   :target: https://www.adhoc.com.ar
+
+.. |icon| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-icon.png
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
    :alt: License: AGPL-3
 
 =============
 Project Stage
 =============
 
-Create module project stage and add it on project header form and project search.
+This module create class project stage and add it on project header form and project search.
 
 Installation
 ============
 
```

Comment:
This sections are empty, will be good if they have content as it is in the README guideline template.
Would be good if could look like the next one:

![image](https://user-images.githubusercontent.com/7593953/41478402-eb611cda-709d-11e8-8c6e-907118962a54.png)